app/views/find_page.py

```diff
@@ -14,17 +14,33 @@
 @find_page_bp.route("/find-page/invite")
 @confirmed_required
 def find_page_invite():
-    date_request_data = suggest_matches(db.session, current_user.id, 1)
-    if not date_request_data:
-        return render_template("main/find_page.html", date_request_data=None)
-
     date_request_form = DateRequestForm()
     is_requesting = True
-    date_request_data = date_request_data[0].profile
+    
+    date_request_data = get_suggested_matches_data(10)
+    if not date_request_data:
+        date_request_data=None
+        
     return render_template(
         "main/find_page.html", date_request_data=date_request_data, form=date_request_form, is_requesting=is_requesting
     )
```

Comment:
A nie możemy przesłać pustej listy do frontendu? Nie będzie to wtedy działać?

Reply:
Zrobiłem tak bo chyba Olek tak zrobił że przekazywał None i wtedy na podstawie tego wyświetlał w jinjy albo propozycje albo komunikat o braku. Ale jeśli jinja ewaluuje pustą listę jako false to powinno działać też z listą.

Reply:
Z pustą listą u mnie wydaje mi się że też działa